doc/authoring_command_modules/README.md

```diff
@@ -141,38 +155,30 @@ Testing
 Run all tests in a module:
 
 ```
-run_tests --module <module>
-OR
-python -m unittest discover -s <path_to_your_command_module>/tests
+run_tests --module <module> [--live] [--parallel]
 ```
 
 Run an individual test:
 
 ```
-python <path_to_your_command_module>/<file> <class name>
+run_tests --module <module> --test <file>.<class>[.<test>]
 ```
-For example `python src/command_modules/azure-cli-appservice/tests/test_webapp_commands.py WebappBasicE2ETest`
+For example `run_tests --module example --test test_example.ExampleTests.test_example`
 
-Note:  
-The following is required in the test file when running an individual test.  
-```
-if __name__ == '__main__':
-    unittest.main()
-```
 
-PyLint
-------
+Style Checks
+------------
 
 ```
-pylint -r n <path_to_your_command_module>/azure
+check_style --module <module> [--pylint] [--pep8]
```

Comment:
Do we plan on deprecating `check_style`?
If so, we should provide the alternative in docs. If not, we can just keep it and remove the deprecation message.

```
$ check_style --help
The check_style command is going to be replaced by 'azdev style' command.
```

Reply:
Good point.